src/main/java/org/apache/commons/compress/archivers/cpio/CpioArchiveEntry.java

```diff
@@ -565,7 +565,12 @@ public int getHeaderPadCount(final long nameSize) {
         }
         int size = this.headerSize + 1; // Name has terminating null
         if (name != null) {
-            size = ExactMath.add(size, nameSize);
+            try {
+                size = ExactMath.add(size, nameSize);
```

Comment:
There a dozen call sites of `ExactMath#add()`; surely it would be redundant to duplicate this pattern. Why not create a wrapper method in `ExactMath`?